stac_api/runtime/src/app.py

```diff
@@ -85,7 +87,15 @@ async def lifespan(app: FastAPI):
     middlewares=[Middleware(CompressionMiddleware), Middleware(ValidationMiddleware)],
```

Comment:
@botanical STAC Auth Proxy apply this same compression middleware but at the end of the request chain.  As such, we don't want it here.

```suggestion
    middlewares=[Middleware(ValidationMiddleware)],
```